design-reviews/recipes/engagement-process.md

```diff
@@ -0,0 +1,129 @@
+
+## Table of Contents
+
+- [Envisioning / ADS](#envisioning-/-ads)
+- [Game Plan](#game-plan)
+- [Sprint Planning](#sprint-planning)
+- [Sprint Backlog Grooming](#sprint-backlog-grooming)
+- [Sprint Retrospectives](#sprint-retrospectives)
+- [Wrap-up Sprints](#wrap-up-sprints)
+
+## Envisioning / ADS
+
+Early on within an engagement process we are working with customers to understand their unique goals and objectives.  We are defining requirements to understand definition of done.  We are diving deep into their existing infrastructure and architecture to understand constraints.
+
+During this time we are uncovering as many unknowns as possible to help us jointly generate a design to meet the customers goals.  We are leveraging all the new found information an incorporating it into our game plan design for the customer engagement.
+
+> **Tip**: We have not uncovered all the unknowns at this point. You will continue to learn and discover more during the lifetime of your engagement
+
+## Game Plan
+
+The game plan document is our guiding star throughout the engagement.  The game plan document should include the following sections and answer the following questions.
+
+### Deliverable
+
+1. What are we delivering?
+1. What does success look like?
+1. What is the definition of done?
+1. What security, privacy and/or performance requirements are required?
+
+### Design
+
+1. What does the high level architecture look like?
+1. What does the technology stack look like?
+1. How does this solution fit into the customers overall product / service?
+
+### Process
+
+1. How will engineering fundamentals be leveraged during the project?
+1. What is the proposed sprint schedule?  How many sprints?
+1. Who is the team? CSE engineers? Customer Engineers?
+1. What is the sharing plan after the engagement is complete?
+
+### Ready to start
+
+1. Has the customer signed off on the plan?
+1. Are there any blocking issues to starting the engagement?
+1. What are the known risks?
+
+> **Tip**: Keep your game plan up to date as your design changes over time
+
+## Checkpoint Reviews
+
+Our monthly checkpoints are a great way for the team to check-in with leadership and other CSE SME's and provide a status update to the larger team.
+
+Some key items to discuss:
+
+### Design Reviews
+
+1. Present any key design reviews created since the last checkpoint or game plan
+1. Present any new design artifacts including API contracts, architecture diagrams, workflows, deployment pipelines, etc.
+1. Describe if there are any high level architecture or design changes since last review
+1. How have your design changes impacted the initial game plan
+
+## Sprint Planning
+
+In many of our engagements we will be working with our customers using a SCRUM agile development process.  Sprint planning is a great opportunity to dive deep into the next set of high priority work.  Some key points to address are the following:

```

Comment:
Maybe linking Sprint planning to https://github.com/microsoft/code-with-engineering-playbook/tree/master/sprint-planning

Reply:
There is also a section on Stand-Ups https://github.com/microsoft/code-with-engineering-playbook/tree/master/stand-ups and Scrum-of-Scrums https://github.com/microsoft/code-with-engineering-playbook/tree/master/scrum-of-scrums